README.rst

```diff
@@ -41,11 +41,11 @@ Windows
 - ``git clone https://github.com/epcpower/m-pm``
 - ``cd m-pm``
 - ``git submodule update --init``
-- ``poetry install``
-- ``poetry run builduimpm``
-- ``poetry run builduiepyqlib``
+- ``poetry@1.5.1 install``
+- ``poetry@1.5.1 run builduipm``
```

Comment:
Be very careful here...
```suggestion
- ``poetry@1.5.1 run builduimpm``
```

Reply:
[08d62ce](https://github.com/epcpower/m-pm/pull/6/commits/08d62ced08a29f8e813387fac08480f0519166a0)